### PR TITLE
Add minHour and maxHour to be displayed in day/week view

### PR DESCRIPTION
**Pull request by adambinnersley**

### CODE REVIEW

File: src/components/CalendarBody.tsx [hunk 1]
```diff
@@ -100,6 +103,9 @@ function _CalendarBody<T extends ICalendarEventBase>({
 }: CalendarBodyProps<T>) {
   const scrollView = React.useRef<ScrollView>(null)
   const { now } = useNow(!hideNowIndicator)
+  const hours = minHour < maxHour && minHour >= 0 && maxHour <= 23
+      ? Array.from({ length: maxHour - minHour + 1 }, (_, i) => minHour + i)
+      : Array.from(Array(24).keys())
```
Comment (acro5piano) [inline src/components/CalendarBody.tsx:108]: Can you validate it in `CalendarContainer.tsx`? like

```ts
if (minHour >= maxHour) { 
  throw new Error('minHour should be greater than maxHour')
}
if (minHour <= 0) { 
  // other error message
}
// other checks
```

I think it's more intuitive to provide developers comprehensive error message than defaulting the basic.

File: README.md [hunk 2]
```diff
@@ -161,6 +163,8 @@ export interface CalendarProps<T extends ICalendarEventBase> {
...
+| `minHour`                  | no       | `number`                                 | The first hour that will be displayed in week/day modes.                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| `maxHour`                  | no       | `number`                                 | The final hour that will be displayed in week/day modes.                                                                                                                                                                                                                                                                                                                                                                                                                        |
```
Comment (acro5piano) [inline README.md:167]: Thanks,

If you provide examples in the storybook, it's helpful (not mandatory though)

**Review by acro5piano — APPROVED** [review]
Thank you very much! Some minor tweaks are needed but it's almost there.

**Review by acro5piano — CHANGES_REQUESTED** [review]
Sorry I wanted to request changes. misclicked!

**Comment by adambinnersley** [comment]
I've updated to fork to now throw errors on incorrect prop values and added a story to demonstate these new props

**Comment by acro5piano** [comment]
@adambinnersley Fantasitc! Thank you for your great work!

**Comment by jangminhyeok** [comment]
#1053 Hi, There is a problem with event cell position. It happens in iOS 17.4 / iPhone 15 Pro Max. By the way, thank you for your contribution!

**Pull request merged**